dist/js/apps/events-page/app.js

```diff
@@ -280,6 +315,58 @@ var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol
 })(angular.module('eventsPageApp'));
 'use strict';
 
+(function (app) {
+	'use strict';
+
+	var EventRegistrationCtrl = function EventsPageCtrl($window, $scope, $routeParams, eventsService, registrationService, dateUtilityService) {
+		var id = $routeParams.id;
+
+		var vm = this;
+
+		vm.isGroup = 'false';
+		vm.isSubmitted = false;
+		vm.isLoadingResults = false;
+
+		vm.submitHandler = function () {
+			vm.isLoadingResults = true;
+
+			var postModel = {
```

Comment:
Are there going to be more keys on this object? If no,t you could probably remove the outer

Reply:
That's how the vendor API wants it, wrapped in another object. However, you have a point. I just have to make some small changes to the web service to match.

Reply:
10-4